test/RestrictedNORI-transfer.test.ts

```diff
@@ -0,0 +1,531 @@
+import { BigNumber } from 'ethers';
+
+import {
+  expect,
+  advanceTime,
+  batchMintAndListRemovalsForSale,
+} from '@/test/helpers';
+import {
+  setupTestLocal,
+  restrictRemovalProceeds,
+  compareScheduleDetailForAddressStructs,
+  compareScheduleSummaryStructs,
+  SECONDS_IN_10_YEARS,
+} from '@/test/helpers/restricted-nori';
+
+describe('RestrictedNORI transferring', () => {
+  describe('success', () => {
+    describe('safeTransferFrom', () => {
+      // eslint-disable-next-line jest/expect-expect -- assertions are in helper function
```

Comment:
another reason to not conditional expect :shrug: disabling this means that eslint no longer has a way to indicate whether the test actually tests something or not

Reply:
in the mean time can you remove these eslint statements and just add this rule to contracts/.eslintrc.js? ([docs](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/expect-expect.md#options))

```ts
"jest/expect-expect": [
    "error",
    {
      "assertFunctionNames": ["compareScheduleDetailForAddressStructs","compareScheduleSummaryStructs"]
    }
]
```

Reply:
better yet enable it for this file only using the example they have in their docs 
```
/* eslint jest/expect-expect: ["error", { "assertFunctionNames": ["expect", "compareScheduleDetailForAddressStructs", "compareScheduleSummaryStructs"] }] */
```
that way we remove the exception to the rule when we fix/remove the conditional expects

Reply:
maybe i'll just get rid of these comparison utilities and do a field-by-field expect in each test instead

Reply:
enabled for this file. I was going to consider replacing the utility with a bunch of line-by-line expects but I think the better move is to figure out how to do it right in the utility itself or some of the tests are going to be really long and hard to read

Reply:
hmm, maybe worth opening a pr for waffle at some point.